lib/operations/command_v2.js

```diff
@@ -7,7 +7,7 @@ const ReadConcern = require('../read_concern');
 const WriteConcern = require('../write_concern');
 const maxWireVersion = require('../core/utils').maxWireVersion;
 const commandSupportsReadConcern = require('../core/sessions').commandSupportsReadConcern;
-const MongoError = require('../error').MongoError;
+const MongoError = require('../core').MongoError;
```

Comment:
Guessing this got pulled in from master at some point, I'm surprised it never caused an issue before.